File: extensions/panache/hibernate-orm-panache-kotlin/runtime/src/main/java/io/quarkus/hibernate/orm/panache/kotlin/runtime/PanacheKotlinHibernateOrmRecorder.java
package io.quarkus.hibernate.orm.panache.kotlin.runtime;

import java.util.Map;

import io.quarkus.hibernate.orm.panache.common.runtime.AbstractJpaOperations;
import io.quarkus.runtime.annotations.Recorder;

@Recorder
public class PanacheKotlinHibernateOrmRecorder {
    public void setEntityToPersistenceUnit(Map<String, String> entityToPersistenceUnit) {
        AbstractJpaOperations.setEntityToPersistenceUnit(entityToPersistenceUnit);
    }
}

File: extensions/panache/hibernate-orm-panache-kotlin/runtime/src/main/kotlin/io/quarkus/hibernate/orm/panache/kotlin/runtime/PanacheKotlinHibernateOrmRecorder.kt
package io.quarkus.hibernate.orm.panache.kotlin.runtime

import io.quarkus.hibernate.orm.panache.common.runtime.AbstractJpaOperations
import io.quarkus.runtime.annotations.Recorder

@Recorder
open class PanacheKotlinHibernateOrmRecorder {
    open fun setEntityToPersistenceUnit(entityToPersistenceUnit: Map<String?, String?>?) {
        AbstractJpaOperations.setEntityToPersistenceUnit(entityToPersistenceUnit)
    }
}
